opentelemetry-sdk/Cargo.toml

```diff
@@ -17,7 +17,7 @@ futures-channel = "0.3"
 futures-executor = { workspace = true }
 futures-util = { workspace = true, features = ["std", "sink", "async-await-macro"] }
 percent-encoding = { version = "2.0", optional = true }
-rand = { workspace = true, features = ["std", "std_rng","small_rng"], optional = true }
+rand = { workspace = true, features = ["std", "std_rng", "small_rng", "os_rng", "thread_rng"], optional = true }
```

Comment:
I see we actually removed `thread_rng` in other files. Why do we still need this feature?

Reply:
it was just renamed, `thread_rng()` -> `rng()`